client/rpc/dex_client.go

```diff
@@ -214,6 +216,48 @@ func (c *HTTP) GetDepth(tradePair string) (*types.OrderBook, error) {
 	return &ob, nil
 }
 
+func (c *HTTP) GetProposals(status types.ProposalStatus, numLatest int64) ([]types.Proposal, error) {
```

Comment:
Do we need to support pagination for proposals query? It is very possible that there are a lot of proposals on binance chain.

Reply:
> Do we need to support pagination for proposals query? It is very possible that there are a lot of proposals on binance chain.

I hope so, but the server not supported now.